tests/test_p4objects.cpp

```diff
@@ -326,7 +326,7 @@ TEST(P4Objects, HeaderStackArith) {
   ASSERT_NO_THROW(h0_f1.get_int());
 
   if (!WITH_VALGRIND) {
-    ASSERT_DEATH(h0_f2.get_int(), "Assertion .* failed");
-    ASSERT_DEATH(h1_f2.get_int(), "Assertion .* failed");
```

Comment:
clang on the mac prints the assertion as "Assertion failed: (arith) ...", which fails the regex you had before which implies two spaces.